### DISCUSSION TITLE
Has 2.10 been released officially?

### DISCUSSION THREAD

**Discussion by dejantep** [netbox-community/netbox#5476] [Q&A] [answered]
I see work is being done on 2.10.1 and usually i see announcements on Google groups

**Comment by candlerb** [comment]
In Github, if you select `Watch > Custom > Releases` and check it, then you'll get an automatic notification whenever a new release is made.

**Reply by dejantep** [reply]
thanks. So it's released?

**Comment by candlerb** [comment, marked as answer]
It has indeed (as has 2.10.1). https://github.com/netbox-community/netbox/releases/

**Reply by dejantep** [reply]
Just upgraded and wow...So far so good.
Circuit tracing and termination is great improvement IMHO. 
Great job!!

**Comment by chicks-net** [comment]
Are the announcements to google groups broken?

**Reply by jeremystretch** [reply]
Only in that I've stopped sending them. Given that NetBox now has the ability to alert on new releases it seems very redundant. Long-term, I'd actually like to turn down the mailing list in favor of GitHub discussions, to keep the discussion closer to the code itself.
